deploy/catalog_resources/kiecloud-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,201 @@
+          - kind: Secret
+            version: v1
+          - kind: PersistentVolumeClaim
+            version: v1
+          - kind: ServiceAccount
+            version: v1
+          - kind: Role
+            version: authorization.openshift.io/v1
+          - kind: RoleBinding
+            version: authorization.openshift.io/v1
+          - kind: Service
+            version: v1
+          - kind: Route
+            version: route.openshift.io/v1
+        specDescriptors:
+          - description: Environment of PAM to deploy.
+            displayName: Environment
+            path: environment
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:label"
+          - description: The desired number of Kie Server Deployments for the KieApp environment
+            displayName: Kie Deployments
+            path: kieDeployments
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:podCount"
+        statusDescriptors:
+          - description: The status of each of the Deployments for the KieApp environment.
+            displayName: Deployment Status
+            path: deployments
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:podStatuses"
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+        - serviceAccountName: kie-cloud-operator
+          rules:
+            - apiGroups:
+                - ""
+              resources:
+                - "*"
+              verbs:
+                - "*"
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+                - serviceaccounts
+              verbs:
+                - "*"
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - daemonsets
+                - replicasets
+                - statefulsets
+              verbs:
+                - "*"
+            - apiGroups:
+                - app.kiegroup.org
+                - apps.openshift.io
+                - image.openshift.io
+                - build.openshift.io
+                - route.openshift.io
+              resources:
+                - "*"
+              verbs:
+                - "*"
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - roles
+                - rolebindings
+              verbs:
+                - "*"
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+      deployments:
+        - name: kie-cloud-operator
+          spec:
+            replicas: 1
+            strategy:
+              type: Recreate
```

Comment:
Do you mind moving the strategy.type from spec.template.spec to spec in the operator.yaml file as it is here?

Reply:
ya, good catch